dashboard_desc.txt

```diff
@@ -0,0 +1,6 @@
+For building the dashboard, I liked the mockup given and decided to build the project off it. I wanted a minimalist design and so I stuck with a two-tone background, I choose a light green, in homage to Sidekick's green tone.
+
+I used bootstrap 4.0 for their powerful flexbox layout. Each course is it's own row and the credit squares are nested flexboxes.
```

Comment:
If you didn't have a grid framework like Bootstrap, how else might you be able to accomplish this layout?